bndtools.core/src/bndtools/central/Central.java

```diff
@@ -212,6 +215,20 @@ public synchronized static Workspace getWorkspace() throws Exception {
             newWorkspace.addBasicPlugin(new WorkspaceListener(newWorkspace));
             newWorkspace.addBasicPlugin(instance.repoListenerTracker);
             newWorkspace.addBasicPlugin(getWorkspaceR5Repository());
+
+            // need to remove any progress plugins that already exist so we can favor JobProgress
+            Set<Object> progressPlugins = new HashSet<>();
+
+            for (Object plugin : newWorkspace.getPlugins()) {
+                if (plugin instanceof ProgressPlugin) {
+                    progressPlugins.add(plugin);
+                }
+            }
+
+            for (Object plugin : progressPlugins) {
+                newWorkspace.removeBasicPlugin(plugin);
+            }
+
```

Comment:
So in the httpClient it just calls, registry.getPlugin(ProgressPlugin.class) and I assume it would just grab the first one.  So I wanted to make sure JobProgress was the only one.